### PR TITLE
PRO-2073 : Move Liquibase to external task

### PR DESCRIPTION
**Pull request by reddy1praveen**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: liquibase.gradle [hunk 1]
```diff
@@ -0,0 +1,52 @@
+configurations {
+    liquibase
+}
+
+dependencies {
+    liquibase group: 'org.liquibase.ext', name: 'liquibase-hibernate5', version: 3.6
+}
+
+//loading properties file.
```
Comment (nigeldunne) [inline liquibase.gradle]: remove this comment, it's obvious

File: src/main/resources/liquibase.properties [hunk 2]
```diff
@@ -2,5 +2,11 @@ changeLogFile=classpath:db/changelog/db.changelog-master.yaml
 url=jdbc:postgresql://localhost:15432/probate
 
 driver=org.postgresql.Driver
-outputChangeLogFile=src/main/resources/liquibase/liquibase-outputChangeLog.yml
+username=probate
+password=password123
 
+# liquibase:generateChangeLog
```
Comment (nigeldunne) [inline src/main/resources/liquibase.properties]: remove these comments

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -40,3 +40,5 @@ logging:
 azure:
   app_insights_key: ${APPINSIGHTS_INSTRUMENTATIONKEY:false}
 
+liquibase:
```
Comment (nigeldunne) [inline src/main/resources/application.yml]: Better to exclude it from the jar in the first place

File: build.gradle [hunk 4]
```diff
@@ -160,9 +160,8 @@ dependencies {
         exclude module: 'slf4j-simple'
     }
 
-//    compile group: 'org.liquibase.ext', name:'liquibase-hibernate5', version: '3.6'
-//    compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.logging
-    compile group: 'org.liquibase.ext', name:'liquibase-hibernate5', version: '3.6'
+    compile group: 'org.liquibase', name: 'liquibase-core', version: '3.5.5'
```
Comment (nigeldunne) [inline build.gradle]: Why not use the latest?

Comment (reddy1praveen) [reply build.gradle]: using 3.6.1

File: src/main/resources/application.yml (hunk 3 above)
Comment (reddy1praveen) [reply src/main/resources/application.yml]: may need it locally

File: liquibase.gradle (hunk 1 above)
Comment (reddy1praveen) [reply liquibase.gradle]: removed

File: src/main/resources/liquibase.properties (hunk 2 above)
Comment (reddy1praveen) [reply src/main/resources/liquibase.properties]: removed

**Pull request merged**